{
  "contest": 145,
  "handle": "Deivitto",
  "address": "0xa322bD417154D04d5704952E550288f0f91044Bd",
  "risk": "2",
  "title": "Unexpected assert() reached by the code",
  "issueId": 287,
  "issueUrl": "https://github.com/code-423n4/2022-07-ens-findings/issues/287"
}